tests/modules/test_proxy2jmx.py

```diff
@@ -36,7 +52,7 @@ def test_no_token(self):
     def test_full(self):
         self.obj.api_delay = 1
         self.obj.responses = [
-            ResponseEmul(404, ''),
```

Comment:
It was true for the old interface (/mitmproxies). New (/proxy) doesn't return 404 in normal cases. So it's not necessary to check for 404 if we don't verify especially the old interface.